src/helpers/git_helpers/branch_helpers.sh

```diff
@@ -39,6 +40,17 @@ function delete_remote_only_branch {
 }
 
 
+# Exits if the repository does not have a branch with the given name
+function ensure_does_have_branch {
```

Comment:
A hurray to proper function naming! A method with that name already exists, and its does the same thing!  :)